src/frontend/src/lib/canisters/backend.errors.ts

```diff
@@ -0,0 +1,27 @@
+import type {
+	BtcAddPendingTransactionError,
+	SelectedUtxosFeeError
+} from '$declarations/backend/backend.did';
+import { CanisterInternalError } from '$lib/canisters/errors';
+
+export const btcPendingTransactionError = (response: BtcAddPendingTransactionError) => {
+	if ('InternalError' in response) {
+		return new CanisterInternalError(response.InternalError.msg);
+	}
+
+	return new Error('Unknown BtcAddPendingTransactionError');
```

Comment:
I would throw a particular type of error here as well. It can be `CanisterInternalError` or another type but, not using a generic `Error` can be interesting at some point. Also reflects the pattern we have in ic-js.

Reply:
Agreed 👍